ipset_linux.go

```diff
@@ -0,0 +1,288 @@
+package netlink
+
+import (
+	"log"
+	"net"
+	"syscall"
+
+	"github.com/vishvananda/netlink/nl"
+	"golang.org/x/sys/unix"
+)
+
+type IPSetEntry struct {
```

Comment:
Exported name without docstring.
Please add docstrings to all names which are started with upper case, using convention described in https://blog.golang.org/godoc-documenting-go-code

Reply:
done